src/main/kotlin/com/github/aivancioglo/resttest/http/Response.kt

```diff
@@ -101,24 +133,50 @@ abstract class Response() {
     }
 
     /**
-     * For getting responseSpecification code of last responseSpecification.
+     * Set root path.
```

Comment:
You are following the "Making" word everywhere and not it is "Set", "Add". Please make it follow one style.